web-client/src/client/bls_cache.rs

```diff
@@ -0,0 +1,89 @@
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize, Compress};
+use idb::{Database, Error, KeyPath, ObjectStore, TransactionMode};
+use nimiq_bls::{G2Projective, LazyPublicKey, PublicKey};
+use nimiq_serde::{Deserialize, Serialize};
+
+/// Caches decompressed BlsPublicKeys in an IndexedDB
+pub(crate) struct BlsCache {
+    db: Option<Database>,
+    keys: Vec<LazyPublicKey>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+struct BlsKeyEntry {
+    compressed_key: String,
+    public_key: String,
```

Comment:
We talked about adding a "last used" field so we can apply some sort of cache eviction in the future.